README.md

```diff
@@ -60,6 +60,28 @@ The repository can now be considered a "holding bay" for code developed by Neuro
 - cd into the repo: `cd brainlit`
 - install brainlit: `pip install -e .`
 
+### For Windows Users setting up a Conda environment:
+
+Users currently may run into an issue with installing dependencies on Python 3.8. There are a couple workarounds currently available:
+
+#### Use Python 3.7 - RECOMMENDED
+
+- Create a new environment using Python 3.7 instead: `conda create --name brainlit3.7 python=3.7`
+
+- Switch to the following branch using `git checkout python3.7-experimental` in the Anaconda prompt
+
```

Comment:
Users no longer need to do this, the default branch (deploy) is now python 3.7 compatible (PR [here](https://github.com/NeuroDataDesign/brainlit/pull/7)).